File: HomeMart/src/app/_layout/home/vattumodel.ts
export class VatTu{
    public  TenVatTu : string;
    public  MaVatTu :string;
    public  DonGia : number;
    public  SoTon :number;
    public SoLuong : number;
    public  HinhAnh :Array<string>;
    public ItemTotal: number;
    public Avartar : Array<number>;
    public selectFavorite : number;
    public DonGiaKhuyenMai :number;
    public TyLeKhuyeMai : number;
}

export class VatTuDTO{
    public Data : Array<VatTu>;
    public ItemTotal:number;
    public PageSize : number;
    public PageNumber : number;
}

export class VatTuDetail{
    public  TenVatTu : string;
    public  MaVatTu :string;
    public  DonGia : number;
    public  SoTon :number;
    public  HinhAnhs :Array<string>;
    public ItemTotal: number;
    public Avartar : Array<number>;
    public selectFavorite : number;
    public  MoTa:string;
    public  NhaCungCap:string;
    public Size : Array<string>;
    public  MaDonVi:string;
    constructor(){
        this.TenVatTu = "";
        this.MaVatTu = "";
        this.DonGia = 0;
        this.SoTon = 0;
        this.Avartar = null;
    }
}

File: BT.API.HOME/BT.API.HOME/Models/VatTuModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BT.API.HOME.Models
{
    public class VatTuModel
    {
        public string TenVatTu { get; set; }
        public string MaVatTu { get; set; }
        public decimal DonGia { get; set; }
        public decimal SoTon { get; set; }
        public List<string> HinhAnh { get; set; }
        public byte[] Avatar { get; set; }
        public decimal DonGiaKhuyenMai { get; set; }
        public decimal TyLeKhuyeMai { get; set; }
    }

    public class VatTuDTO
    {
        public List<VatTuModel> Data { get; set; }
        public decimal ItemTotal { get; set; }
        public decimal PageSize { get; set; }
        public decimal PageNumber { get; set; }
        public VatTuDTO(List<VatTuModel> _data)
        {
            Data = _data;
        }
    }

    public class VatTuDetail
    {
        public string MaVatTu { get; set; }
        public string TenVatTu { get; set; }
        public string MoTa { get; set; }
        public string MaNhomVatTu { get; set; }
        public decimal DonGia { get; set; }
        public decimal SoTon { get; set; }
        public string NhaCungCap { get; set; }
        public string[] Size { get; set; }
        public byte[] Avatar { get; set; }
        public List<string> HinhAnhs { get; set; }
        public string MaDonVi { get; set; }
    }
}